repose-aggregator/src/docs/asciidoc/recipes/docker.adoc

```diff
@@ -55,7 +55,7 @@ Let's break that command down and take a closer look at what it is doing:
 <1> This is the part of the command that tells Docker to create and start a container.
 <2> This option will run the container in detached mode.
     In detached mode, the container will run in the background.
-    If we were to run in attached mode (the default) instead of detached mode, the terminal we used to execute this command would receive all of the output (i.e., STDIN, STDOUT, and STDERR) from the container.
+    If we were to run in the default attached mode instead of optional detached mode, the terminal we used to execute this command would receive all of the container output (e.g. STDOUT, and STDERR).
```

Comment:
I find "optional detached mode" to be slightly awkward phrasing. Really, including "optional" is redundant, so I would probably just drop that word.

Oh, and I think `i.e.` fits better here than `e.g.` since the enumerated values are exhaustive. That is, I would replace:
> (e.g. STDOUT, and STDERR).

with:

> (i.e., `STDOUT` and `STDERR`).

You may notice that I would also drop the comma before the "and" and include a comma after "i.e." because I believe that is more correct and also because it is consistent with the rest of this document. [On correctness](https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/).

Oh, and I formatted STDOUT and STDERR as code just because that is what the Docker folks do in their docs.

Reply:
I also agree with using `i.e.` over `e.g.` and reluctantly acknowledge our standardization on using the following comma (i.e. using `i.e.,` 😉  ).

Reply:
Updated.

Reply:
This one slipped passed me. I updated this in a couple of other places.

Reply:
Worse than that, I changed this one to be wrong.